pages/in/widget/iscn/index.vue

```diff
@@ -374,6 +374,7 @@ export default {
           url,
         } = this;
         const txHash = await this.sendISCNSignature({
+          cosmosWallet,
           userId: this.getUserId,
           displayName: this.getUserInfo.displayName,
           from,
```

Comment:
Is `from` or `cosmosWallet` actually used in this function?

Reply:
it's cosmosWallet, will change